README.md

```diff
@@ -6,17 +6,33 @@ The purpose of this controller is to ensure that profiles/namespaces are correct
 
 The sections below detail specific labels and the logic they use.
 
-## Employee Only Features
+The initial implementation involved the bundling of features into either employee or non-employee features, but the addition of exception lists introduced the requirement for more fine-grain discretionary access control. This means we have a label for each feature.
+## SAS notebook feature
 
-This controller adds the `state.aaw.statcan.gc.ca/employee-only-features` label to the Profiles based on the Pods in that namespace.
-If any Pods in that namespace are using a SAS image, it will set the label in the Profile to `true`, otherwise `false`.
+This controller adds the `state.aaw.statcan.gc.ca/hasSasNotebookFeature` and `state.aaw.statcan.gc.ca/existsNonSasUser` labels to the Profiles based on the Pods and users in that namespace.
```

Comment:
I think this should be separated by [dashes](https://github.com/StatCan/aaw-profile-state-controller/blob/f455fa3c25580675bb3a10b7b9308425a886e11b/pkg/controller/handler.go#L19) and not camelCase right? Also maybe should be `exists-non-sas-notebook-user` right.